mocktracer/CMakeLists.txt

```diff
@@ -18,7 +18,9 @@ if (BUILD_SHARED_LIBS)
                                                SOVERSION ${OPENTRACING_VERSION_MAJOR})
   target_link_libraries(opentracing_mocktracer PUBLIC opentracing)
   target_compile_definitions(opentracing_mocktracer PRIVATE OPENTRACING_MOCK_TRACER_EXPORTS)
-  install(TARGETS opentracing_mocktracer EXPORT OpenTracingTargets
+  install(TARGETS opentracing_mocktracer
+          COMPONENT DIST
+          EXPORT OpenTracingTargets
```

Comment:
Should the headers for the mocktracer (below) be installed into the DEVEL component?

Reply:
I had no use for them. They could be installed too but it would have to be in a subfolder because some share names that would overwrite other files.

Reply:
You can still use the mocktracer without them, but only from the dynamic loading interface.

All the headers should already be in the mocktracer folder so they shouldn't clash.

Up to you whether you want to install them or not, I'm ok either way.

Reply:
I'm fine without them.